### PR TITLE
Design layout and login page

### LINKED ISSUE #3: Design Initial area component

### PR DESCRIPTION
**Pull request by anthonyminyungi**
- It closes #3 
- This PR contains the Layout and Login page component.

### CODE REVIEW

File: src/components/login/index.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Typography, TextField, Button, makeStyles } from '@material-ui/core';
+import LockOpenIcon from '@material-ui/icons/LockOpen';
+
+import Layout from '../layout';
+
+type LoginPageProps = {
+    // TODO: specify properties and their types
+};
```
Comment (anthonyminyungi) [inline src/components/login/index.tsx:9]: This type will be specified later.

File: src/components/login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}));
+
+// TODO: specify properties and their types
+export default function LoginPage() {
+    const classes = useStyles();
+
+    return (
+        <Layout>
+            <div className={classes.topSpacing} />
+            <Typography className={classes.head} variant="h1">
```
Comment (anthonyminyungi) [inline src/components/login/index.tsx:55]: Currently, the codes have a 4 space indentation, it should be fixed later when we apply `eslint` onto this project.

File: src/components/login/index.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Typography, TextField, Button, makeStyles } from '@material-ui/core';
+import LockOpenIcon from '@material-ui/icons/LockOpen';
+
+import Layout from '../layout';
+
+type LoginPageProps = {
+    // TODO: specify properties and their types
+};
+
```
Comment (anthonyminyungi) [inline src/components/login/index.tsx:10]: This type will be specified later.

File: src/App.tsx [hunk 4]
```diff
@@ -1,26 +1,11 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <LoginPage />
```
Comment (anthonyminyungi) [inline src/App.tsx:8]: Routing will be applied in `App.tsx` component later with `react-router-dom`.

File: src/components/layout/index.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+}));
+
+export default function Layout ({ children }: LayoutProps) {
+    const classes = useStyles();
+    return (
+        <Container className={classes.container}>
+            {children}
+        </Container>
+    );
+};
```
Comment (anthonyminyungi) [inline src/components/layout/index.tsx:27]: I'd like to call it an `abstract` component, but I'm not sure this kind of component has its name.
This component will be reused per every `page` component.

**Review by anthonyminyungi — COMMENTED** [review]
Self-review

**Review by HyeockJinKim — APPROVED** [review]
LGTM. It's great.

**Pull request merged**